File: Python/8py 1/check_alive.py
"""
|--------------------------------------------------------------------------
| Grasshopper - If/else syntax debug
|--------------------------------------------------------------------------
|
"""

def check_alive(health):
   return health > 0


check_alive = (0).__lt__


def check_alive(health):
   if (health > 0):
      return True
   else:
      return False


def check_alive(health):
   return True if health > 0 else False


print(check_alive(5), True)
print(check_alive(0), False)
print(check_alive(-5), False)

File: JavaScript/8js 1/checkAlive.js
/*
|--------------------------------------------------------------------------
| Grasshopper - If/else syntax debug
|--------------------------------------------------------------------------
|
*/

function checkAlive(health) {
   return health > 0;
}


function checkAlive(health) {
   return health > 0 ? true : false;
}


function checkAlive(health) {
   if (health > 0) {
      return true;
   } else {
      return false;
   }
}


console.log(checkAlive(5), true)
console.log(checkAlive(0), false)